scenes/enumerated/problem_12674/english_text.py

```diff
@@ -0,0 +1,12 @@
+TASK_NUMBER_STR = "P. 12674"
+LEFT_CAR_SPEAD = r"80\text{ km/h}"
+RIGHT_CAR_SPEAD = r"70\text{ km/h}"
+MEETING_MOMENT = r"\text{Meeting moment}"
+BEFORE_MEETING_1 = r"\text{}"
+BEFORE_MEETING_2 = r"\text{ minutes before the meeting}"
+ONE_HOUR_ROAD = r"\text{ An hour's drive}"
```

Comment:
I think its "an hour drive". Like a "light year", it's distance measurement, not time.